libraries/AP_Logger/LogStructure.h

```diff
@@ -1171,6 +1172,7 @@ struct PACKED log_VER {
 // @Field: CHeight: Vehicle height above terrain
 // @Field: Pending: Number of tile requests outstanding
 // @Field: Loaded: Number of tiles in memory
+// @Field: HCorr: terrain correction for arming altitude
```

Comment:
Here too let's stick with "reference offset" instead of "terrain correction"

Reply:
done